### DISCUSSION TITLE
Automatic Deployment via Git Version Control (cPanel)

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#11905] [General - Components / Directives / etc] [answered]
Can I achieve automatic deployment of Quasar SPA via cPanel's Git Version Control - https://www.youtube.com/watch?v=-whBU90olDA? If not, what's the best solution on this. My app is a combination of:

1. Front End - Quasar Framework (version 1)
2. Back End - Express.js
3. Database - MongoDB

**Comment by hawkeye64** [comment, marked as answer]
I think this is a question to ask the cPanel people.

**Reply by susansiow** [reply]
Looks like DigitalOcean App Platform is a better choice, thanks!
